lib/prepare.js

```diff
@@ -335,6 +324,26 @@ function sort (arr) {
   })
 }
 
+async function resolveConfig (dir) {
+  const configPath = path.resolve(dir, 'config.js')
+  const configYmlPath = path.resolve(dir, 'config.yml')
+  const configTomlPath = path.resolve(dir, 'config.toml')
+
+  delete require.cache[configPath]
+
+  // resolve siteConfig
+  let config = {}
```

Comment:
Just my honest opinion 😊❤️🙈, I think 335-344 would be cleaner with the following: 

```js
if (fs.existsSync(configYmlPath)) {
  return await parseConfig(configYmlPath)
}

if (fs.existsSync(configTomlPath)) {
  return await parseConfig(configTomlPath)
}

if (fs.existsSync(configPath)) {
  return require(configPath)
}

return {}
```

Reply:
Uh looks better. I just moved things down into a functions without much refactor.

Reply:
Yea yea! Just spot that when I see the changes, might be worth it to make the change.